### PR TITLE
add Sell Complete Sets and Dispute notifications

### PR DESCRIPTION
**Pull request by andrewdamelio**
### Add dispute/sell full shares notifications

@bthaile For showing disputeInfo for position holder users, I had to add ```dispatch(loadDisputing());``` to ```load-account-data.js```. I didn't notice a performance hit with this but please let me know if there is a more performant way to go about this.

### CODE REVIEW

File: src/modules/account/components/notifications/notifications.tsx [hunk 1]
```diff
@@ -29,16 +33,20 @@ class Notifications extends React.Component<NotificationsProps> {
...
           this.updateState(nextProps);
     }
   }
 
   updateState(props: NotificationsProps) {
     this.props.updateNotifications(
-      this.generateCards(this.props.resolvedMarketsOpenOrders, 'resolvedMarketsOpenOrders')
-      .concat(this.generateCards(this.props.reportOnMarkets, 'reportOnMarkets'))
-      .concat(this.generateCards(this.props.finalizedMarkets, 'finalizedMarkets')));
+      this.generateCards(props.resolvedMarketsOpenOrders, 'resolvedMarketsOpenOrders')
```
Comment (bthaile) [inline src/modules/account/components/notifications/notifications.tsx]: 'resolvedMarketsOpenOrders' like these can be put in constants as NOTIFICATION_TYPES: {....}

I'd like to remove magic strings.

File: src/modules/notifications/selectors/notification-state.js [hunk 2]
```diff
@@ -49,6 +49,51 @@ export const selectFinalizeMarkets = createSelector(
...
+  }
+);
+
+// Get all markets in dispute, for market creators and user with positions in disputed markets
+export const selectMarketsInDispute = createSelector(
+  selectMarkets,
+  selectLoginAccountAddress,
+  (markets, address) => {
+    if (markets.length > 0) {
+      return markets
```
Comment (bthaile) [inline src/modules/notifications/selectors/notification-state.js]: are all markets in dispute been loaded?

I think there is a call to get all markets that user has staked on, that would give us the marketIds then we can load them.

File: src/modules/account/components/notifications/notifications.tsx (hunk 1 above)
Comment (andrewdamelio) [reply src/modules/account/components/notifications/notifications.tsx]: 👍 on it

File: src/modules/notifications/selectors/notification-state.js (hunk 2 above)
Comment (bthaile) [reply src/modules/notifications/selectors/notification-state.js]: ok, understood on needing to loadDisputing

Comment (andrewdamelio) [reply src/modules/notifications/selectors/notification-state.js]: ⚡️

**Pull request merged**